config/locales/en.yml

```diff
@@ -20,10 +20,32 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
   time:
     formats:
+      default: "%-d %B %Y %H:%M:%S %Z"
+      short: "%-d %b %Y %H:%M %Z"
       tariff: "%d/%m/%Y %H:%M:%S %Z"
+
   date:
     formats:
+      default: "%-d %M %Y"
+      short: "%-d %m %Y"
       tariff: "%d/%m/%Y"
```

Comment:
Can we remove the tariff locales?

Reply:
These are GovUK patterns ones so need to keep them